client/dashboard/sites/backups/backup-notices.tsx

```diff
@@ -0,0 +1,75 @@
+import { JETPACK_CONTACT_SUPPORT } from '@automattic/urls';
+import { Button, ExternalLink } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import { useBackupState } from '../../app/hooks/site-backup-state';
+import { useFormattedTime } from '../../components/formatted-time';
+import { Notice } from '../../components/notice';
+import type { Site } from '../../data/types';
+
+/**
+ * Renders a contextual Notice based on the site's backup status
+ */
+export function BackupNotices( { site }: { site: Site } ) {
+	const { status, backup } = useBackupState( site.ID );
+	const backupDate = useFormattedTime( backup?.started ?? '', {
+		timeStyle: 'short',
+	} );
+
+	if ( status === 'running' ) {
+		return (
+			<Notice
+				variant="info"
+				title={ sprintf(
+					/* translators: %d is the backup progress percentage. */
+					__( 'Generating backup… (%d%% progress)' ),
+					backup?.percent ?? 0
+				) }
+			>
+				{ sprintf(
+					/* translators: %s is a date, like "Today at 10:00". */
+					__( 'We’re making a backup of your site from %s' ),
+					backupDate
+				) }
+			</Notice>
+		);
+	}
+
+	if ( status === 'success' ) {
+		return (
+			<Notice variant="success" title={ __( 'Backup completed' ) }>
+				{ __( 'You’ll be able to access your new backup in just a few minutes.' ) }
```

Comment:
The design copy says `minutes`, but maybe we can use `moments` instead? It usually only takes a few seconds for the new backup to be available.

```suggestion
				{ __( 'You’ll be able to access your new backup in just a few moments.' ) }
```

Reply:
I think it makes sense, we were just [reusing the one from the older component](https://github.com/Automattic/wp-calypso/blob/b8fb9b6d384195537fdb2ac954df7f6c69cc4933/client/components/jetpack/daily-backup-status/status-card/backup-just-completed.tsx#L43).

Reply:
@ederrengifo does this copy update make sense to you? If so, I can address it in a follow-up PR.